coalib/results/AbsolutePosition.py

```diff
@@ -0,0 +1,58 @@
+from coalib.results.TextPosition import TextPosition
+from coalib.misc.Decorators import enforce_signature
+
+
+class AbsolutePosition(TextPosition):
+
+    @enforce_signature
+    def __init__(self,
+                 text: (tuple, list, None)=None,
+                 position: (int, None)=None):
+        """
+        Creates an AbsolutePosition object that represents the index of a
+        character in a string.
+        
```

Comment:
Which version of coala do you have ? maybe this is a bug that has creeped in ? Can you test more ?